drivers/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ add_subdirectory(hw_cc3xx)
 if (CONFIG_MPSL AND NOT CONFIG_MPSL_FEM_ONLY)
   add_subdirectory(mpsl)
 endif()
+add_subdirectory_ifdef(CONFIG_MSPI mspi)
```

Comment:
Why this is added only if CONFIG_MSPI is enabled, while gpio is included by default? Is it because gpio should be enabled by default by Zephyr rules? @masz-nordic

Reply:
Because the SDP implementation depends on MSPI KConfigs and structures. So MSPI must be enabled.